### ISSUE TITLE
Notifications - There is a notification about an item found but tapping on it the app shows nothing (beacon with wrong url)

### ISSUE THREAD

**Issue by isabelrios** [mozilla-magnet/magnet-client#180]
We had this issue #70, when the url set is incorrect nothing is shown in the app.

Now with the notifications this can be confusing. I had one beacon incorreclty set and could see '1 items near you' but tapping on the notification the app is open with the Scanning and then Nothing found screen.

Not sure if we could show something like wrong url or someting like a label below the tile saying loading...as can be seen in physical web app

**Comment by wilsonpage** [comment]
This could be addressed when we implement the 'hidden' tiles list, which could contain bad urls. The reason we show the notification is because we have decided not to make network requests from the background notification service, to save on battery and data; therefore we don't know if a found URL is reachable or not.

**Comment by isabelrios** [comment]
Yeah, I agree it could be managed then. But we have to know that we could show that there are # items near you and there are actually one (or more) less.
It is possible to see the physical web notification with a different number at the same time if we have a bad beacon configured.
